textdb/textdb-api/src/main/java/edu/uci/ics/textdb/api/storage/IDataStore.java

```diff
@@ -3,14 +3,16 @@
 import java.util.List;
 
 import edu.uci.ics.textdb.api.common.Attribute;
-import edu.uci.ics.textdb.api.common.ITuple;
+
 
 /**
  * 
  * @author sandeepreddy602
  *
  */
 public interface IDataStore {
-    public void storeData(List<Attribute> schema, List<ITuple> tuples) throws Exception;
-    public void clearData() throws Exception;
+    public void incrementNumDocuments(int incrementBy);
+    public int getNumDocuments();
+    public String getDataDirectory();
+    public List<Attribute> getSchema();

```

Comment:
Later we can introduce a class or interface of "Schema" which is essentially "List<Attribute>". Not for this PR.